specification/cognitiveservices/data-plane/Language/preview/2022-03-01-preview/CustomText-Authoring.json

```diff
@@ -0,0 +1,2871 @@
+          "type": "string"
+        },
+        "lastTrainedDateTime": {
+          "format": "date-time",
+          "description": "Represents the project last trained datetime.",
+          "type": "string"
+        },
+        "lastDeployedDateTime": {
+          "format": "date-time",
+          "description": "Represents the project last deployed datetime.",
+          "type": "string"
+        },
+        "modelKind": {
+          "description": "Gets or sets the new project model kind.",
+          "enum": [
+            "singleLabelClassification",
+            "multiLabelClassification",
+            "namedEntityRecognition"
+          ],
+          "type": "string"
+        },
+        "storageInputContainerName": {
+          "description": "Gets or sets the storage container name.",
+          "type": "string"
+        },
+        "settings": {
+          "description": "Gets or sets the project settings.",
+          "$ref": "#/definitions/ProjectSettings"
+        },
+        "projectName": {
+          "description": "Gets or sets the new project name.",
+          "type": "string"
+        },
+        "multilingual": {
+          "description": "Gets or sets a value indicating whether gets or sets whether the project is multilingual or not.",
+          "type": "boolean"
+        },
+        "description": {
+          "description": "Gets or sets the new project description.",
+          "type": "string"
+        },
+        "language": {
+          "description": "Gets or sets the project language.",
+          "type": "string"
+        }
+      }
+    },
+    "ProjectGetObjectList": {
+      "required": [
+        "value",
+        "nextLink"
+      ],
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Gets or sets the projects.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ProjectGetObject"
+          }
+        },
+        "nextLink": {
+          "description": "Gets or sets the next page link.",
+          "type": "string"
+        }
+      }
+    },
+    "ProjectSettings": {
+      "type": "object"
+    },
+    "SingleLabelClassEvaluationObject": {
+      "required": [
+        "f1",
+        "precision",
+        "recall",
+        "truePositivesCount",
+        "trueNegativesCount",
+        "falsePositivesCount",
+        "falseNegativesCount"
+      ],
+      "type": "object",
+      "properties": {
+        "f1": {
+          "format": "double",
+          "description": "Represents the model precision",
+          "type": "number"
+        },
+        "precision": {
+          "format": "double",
+          "description": "Represents the model recall",
+          "type": "number"
+        },
+        "recall": {
+          "format": "double",
+          "description": "Represents the model F1 score",
+          "type": "number"
+        },
+        "truePositivesCount": {
```

Comment:
>truePositivesCount [](http://example.com/codeflow?start=9&length=18)

board: do we usually specify min/max when they are unlikely to change? Here min=0 seems like a good boundary.